src/execution_tree/primitives/variable.cpp

```diff
@@ -61,6 +62,7 @@ namespace phylanx { namespace execution_tree { namespace primitives
     variable::variable(std::vector<primitive_argument_type>&& operands,
             std::string name)
       : name_(std::move(name))
+      , data_(operands)
```

Comment:
What's your rationale of adding this?

Reply:
The data in operands was being discarded for no apparent reason. I assumed it was a bug.

Reply:
`data_` is properly initialized here: https://github.com/STEllAR-GROUP/phylanx/blob/master/src/execution_tree/primitives/variable.cpp#L75, isn't it?

Reply:
Please remove this change from the proposed changeset as well. The initialization of `data_` here is not necessary if not wrong.